cmd/lakectl/cmd/diff.go

```diff
@@ -54,15 +54,15 @@ var diffCmd = &cobra.Command{
 		if len(args) == diffCmdMinArgs {
 			// got one arg ref: uncommitted changes diff
 			branchURI := MustParseRefURI("ref", args[0])
-			Fmt("Ref: %s\n", branchURI.String())
+			fmt.Printf("Ref: %s\n", branchURI)
```

Comment:
```suggestion
			fmt.Println("Ref: %s", branchURI)
```

Reply:
kept Printf to use %s - we can drop the formatting and switch to use Println, but it is pretty equal.